### PR TITLE
[Chore]: Technical: Translate components/common final part

### PR DESCRIPTION
**Pull request by dariaterekhovaae**
Signed-off-by: Daria Terekhova <daria.terekhova@actionengine.com>

### CODE REVIEW

File: src/components/common/animation-control/animation-controller.ts [hunk 1]
```diff
@@ -47,14 +58,8 @@ function AnimationControllerFactory() {
    * Current time is a point. An array of sorted time steps are provided,
    * animate a moving point jumps to the next step
    */
-  class AnimationController extends Component {
+  class AnimationController extends Component<AnimationControllerProps> {
     // TODO: convert the entire component to use hooks in the next PR
-    static defaultProps = {
```
Comment (heshan0131) [inline src/components/common/animation-control/animation-controller.ts:52]: is there a reason we can't keep `defaultProps`? I find it a lot more sane then adding values wherever we deconstruct `this.props`

File: src/components/common/animation-control/animation-controller.ts [hunk 2]
```diff
@@ -118,8 +124,9 @@ function AnimationControllerFactory() {
     };
 
     _resetAnimtionByTimeStep = () => {
+      const {steps = []} = this.props
```
Comment (heshan0131) [inline src/components/common/animation-control/animation-controller.ts:127]: default value for `steps` should `null`
add below to this function
```js
if (!steps) return;
```

File: src/components/common/animation-control/animation-controller.ts [hunk 3]
```diff
@@ -200,7 +207,7 @@ function AnimationControllerFactory() {
     }
 
     _nextFrameByTimeStep() {
-      const {steps, value} = this.props;
+      const {steps = [], value} = this.props;
```
Comment (heshan0131) [inline src/components/common/animation-control/animation-controller.ts:210]: same as above default value for `steps` should be `null`

File: src/components/common/animation-control/animation-controller.ts (hunk 1 above)
Comment (dariaterekhovaae) [reply src/components/common/animation-control/animation-controller.ts:52]: The problem of defaultProps is that typescript doesn't see it. So sometimes I should duplicate it with deconstruction default values... But I'll keep it for the cases where it's not necessary

**Pull request merged**